From: Sagi Grimberg <sagi@grimberg.me>
Date: Mon, 4 Dec 2017 10:47:10 +0200
Subject: nvmet-rdma: lowering log level for chatty debug messages
Patch-mainline: v4.16-rc1
Git-commit: 424125a09db7a207ab53876db50a6198ca88518f
References: FATE#323952, FATE#322506

It is a bit chatty to report on every deleted queue, so keep it for debug
purposes only.

Signed-off-by: Sagi Grimberg <sagi@grimberg.me>
Reviewed-by: Johannes Thumshirn <jthumshirn@suse.de>
Signed-off-by: Christoph Hellwig <hch@lst.de>
---
 drivers/nvme/target/rdma.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/nvme/target/rdma.c
+++ b/drivers/nvme/target/rdma.c
@@ -976,7 +976,7 @@ static void nvmet_rdma_destroy_queue_ib(
 
 static void nvmet_rdma_free_queue(struct nvmet_rdma_queue *queue)
 {
-	pr_info("freeing queue %d\n", queue->idx);
+	pr_debug("freeing queue %d\n", queue->idx);
 
 	nvmet_sq_destroy(&queue->nvme_sq);
 
